docs/guide/personalization.md

```diff
@@ -265,294 +264,285 @@ For the `content` interface one Content item is returned, for the `contenttypes`
+a notification is sent to https://admin.yoochoose.net (2).
+The recommendation service also notifies other components of the Recommendation engine (3)
+and it eventually fetches the affected content (4) and updates it internally (5).
 
-This mechanism allows the content in the personalization engine to be always in sync with the content on eZ Platform.
+![Subsequent Content exports](img/incremental_content_export.png)
 
-## Enabling recommendations
+## Displaying recommendations
 
 !!! note "Client-based recommendations"
 
-    Recommendations are fetched and rendered asynchronously in the client, so there won't be any additional load on the server.
+    Recommendations are fetched and rendered asynchronously in the client, so there is no additional load on the server.
     Therefore it is crucial to check if the content export has been successful, as e.g. deeplinks and image references are included.
-    If the export is NOT successful, the personalization engine will not have the full content information. This will break the recommendations. Even if the recommendations are displayed, there is a big chance they won't have images, titles or deeplinks.
-
-In order to allow displaying recommendations on your site, you must add some code which will integrate the recommender engine (with your site). This can be achieved with a few steps (assuming that `EzSystemsRecommendlationBundle` is properly configured and enabled in `AppKernel.php`):
-
-Add the following JavaScript assets to your header template:
-
-``` html+twig
-{% javascripts
-    '%kernel.root_dir%/../vendor/components/handlebars.js/handlebars.min.js'
-    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationtemplaterenderer.js'
-    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationtemplatehelper.js'
-    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationrestclient.js'
-%}
-```
+    If the export is NOT successful, the Recommendation engine will not have the full content information.
+    This will break the recommendations. Even if the recommendations are displayed,
+    there is a big chance they won't have images, titles or deeplinks.
 
-Place a dedicated Twig helper in the place where you want to display recommendations:
+In order to display recommendations on your site, use a dedicated `showRecommendationsAction` from the `RecommendationController.php`:
 
 ``` html+twig
-{{ yc_show_recommendations(
-    contentId = content.id,
-    scenario = '',
-    limit = '',
-    contentType = '',
-    template = '',
-    fields = []
-) }}
+render_esi(controller(‘EzRecommendationClientBundle:Recommendation:showRecommendations’, {
+               ‘contextItems’: content.id,
+               ‘scenario’: ‘front’,
+               ‘outputTypeId’: ‘blog_post’,
+               ‘limit’: 3,
+               ‘template’: ‘EzRecommendationClientBundle::recommendations.html.twig’,
+               ‘attributes’: [‘title’, ‘intro’, ‘image’, ‘uri’]
+           }))
 ```
```

Comment:
We also need to add information about required recommendation script 

{% javascripts
    '@EzRecommendationClientBundle/Resources/public/js/EzRecommendationClient.js'
%}

This script must be implemented because it is responsible for send notification ping to recommendation API after click on tracking element